sites/partners/lib/hooks.ts

```diff
@@ -45,42 +50,41 @@ export function useSingleListingData(listingId: string) {
   }
 }
 
-export function useListingsData({
-  page,
-  limit,
-  listingIds,
-  orderBy,
-  orderDir,
-  view = "base",
-}: UseListingsDataProps) {
+export function useListingsData({ page, limit, userId, sort }: UseListingsDataProps) {
   const params = {
     page,
     limit,
-    view,
-    orderBy,
-    orderDir: OrderDirEnum.ASC,
+    view: "base",
+  }
+
+  if (sort) {
+    Object.assign(params, {
+      orderBy: sort?.filter((item) => item.orderBy).map((item) => item.orderBy),
+    })
+    Object.assign(params, {
+      orderDir: sort?.filter((item) => item.orderDir).map((item) => item.orderDir),
+    })
```

Comment:
@emilyjablonski 
If we're not yet brining over the listings order updates in the backend, then we can't yet add this either.